iamproperty/iamproperty.github.io
# Ds 9/audit page

## Summary of Changes:
1. Create a json file that is source controlled, highlighting the file sizes of our assets
2. Create a more visual documentation page to highlight the sizes
3. Re-adjust the config of the builds

## Rationale
Highlight file sizes for each pull request so that we don't accidentally create bloat and affect page performance.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /audit

## Ticket(s)
DS-9
DS-12 - Related

## Checklist

## Notes
The build process for the assets separate from the Vue component library is still a work in progress.
